web/components/input/buttons/RadioGroup/index.js

```diff
@@ -31,7 +36,7 @@ RadioButton.propTypes = {
   // input value for the radio button
   value: PropTypes.string.isRequired,
   // label text for the radio button
-  label: PropTypes.string.isRequired,
+  label: PropTypes.string,
```

Comment:
Could you explain why label is no longer a required prop?

Reply:
oops